frontend/src/pages/AccountSend.vue

```diff
@@ -296,8 +296,13 @@ function useSendForm() {
     const chainId = BigNumber.from(currentChain.value?.chainId).toNumber();
     // Polygon
     if (chainId === 137) {
-      if (isNativeToken(tokenAddress)) return 1.0;
-      else return 3; // stablecoin token minimum
+      if (isNativeToken(tokenAddress)) {
+        return 1.0;
+      } else if (tokenAddress === '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619') {
+        return 0.001; // weth token minimum
+      } else {
+        return 3; // stablecoin token minimum
+      }
```

Comment:
Hmm, we should probably think of a better way to minimum send amounts, since my naive approach here won't scale very well.

Having the relayer's `/tokens?chainId=x` endpoint utilize the `extensions` field of the `TokenInfo` type to return an `extensions.minimumSendAmount` field seems like a good solution

Then for native tokens we'd want specify the minAmount to send in the frontend, perhaps the `nativeCurrency` fields in `models.ts`

Reply:
I agree this probably belongs on the backend. Including it in the tokens endpoint could work. We could also add a field to that endpoint with the the native currency. I think it'd be better to have it all on the backend, rather than split across frontend/backend.

Reply:
The reason I disagree with having the native token also on the backend is because you don't need the relayer/API to send the native token. So if the relayer is ever down, then sending the native token also goes down, which is not really necessary. 

A compromise there is to have the relayer return native token info, but the frontend has its own to fallback on if the relayer is down.

But for the scope of this issue I think this section is fine and we can move this discussion somewhere else!

Reply:
Good point! I like the default + backend override approach. Will create a new issue for tracking and continuing the discussion.

Reply:
https://github.com/ScopeLift/umbra-api/issues/38